File: 0796-rotate-string/0796-rotate-string.py
class Solution(object):
    def rotateString(self, s, goal):
        if len(s) != len(goal):
            return False
        if s==goal:
            return True
        for i in range(1, len(s)):
            ts = s[i:] + s[:i]
            if ts == goal:
                return True
        return False

File: 0796-rotate-string/0796-rotate-string.java
class Solution {
    public boolean rotateString(String s, String goal) {
        if(s.length()!=goal.length()){
            return false;
        }
        if(s.equals(goal)){
            return true;
        }
        for(int i=1;i<s.length();i++){
            String ts = (s.substring(i)+s.substring(0,i));
            if(ts.equals(goal)){
                return true;
            }
        }
        return false;
    }
}